client/src/components/history/history-item.tsx

```diff
@@ -73,15 +77,78 @@ export function ChatItem(props: {
   }
 
   function LinkRenderer(props: { href: string; children: React.ReactNode }) {
-    return (
-      <h3>
-        <a href={props.href} target="_blank" rel="noreferrer">
-          {props.children}
-        </a>
-      </h3>
+    const linkAnswer =
+      props.href.length > 30 ? props.href.slice(0, 30) : props.href;
+    return message.text.includes(prefix) ? (
+      <a
+        href="#"
+        rel="noreferrer"
+        onClick={sentQuestion}
+        data-cy={`question-link-${i}`}
+      >
+        {props.children}
+      </a>
+    ) : (
+      <a
+        href={props.href}
+        target="_blank"
+        rel="noreferrer"
+        onClick={sentQuestion}
+      >
+        {linkAnswer}
+      </a>
     );
   }
 
+  async function sentQuestion() {
+    const REGEX_ASK_LINK = /ask:\/\/display=(.*)/;
+    const answerArray = REGEX_ASK_LINK.exec(message.text);
+    if (answerArray) {
+      const questionSplit = answerArray ? answerArray[1].replace(")", "") : "";
+      const question = questionSplit.split("+").join(" ");
+      await handleAskLinkClicked(question, MentorQuestionSource.USER);
+    }
+    return;
+  }
+
+  const addDelay = (ms: number) => new Promise((res) => setTimeout(res, ms));
+
+  function handleQuestionChanged(
+    question: string,
+    source: MentorQuestionSource
+  ) {
+    dispatch(userInputChanged({ question, source }));
+    return new Promise((res) => {
+      setTimeout(res, 1000);
+    });
+  }
+
+  async function handleQuestionSend(
+    question: string,
+    source: MentorQuestionSource
+  ) {
+    if (!question) {
+      return;
+    }
+    // add animation when question is asked
+    const inputField = document.querySelector("#input-field");
```

Comment:
I think there should be some more `React` way to achieve this in a `useEffect` with `animatingInputField: boolean` as a `useState` hook?

Reply:
OK, I'll work on that